File: patterns/javascript-patterns/pattern84.js
// Pattern 84: ...

//    *
//   * *
//  * * *
// * * * *
//  * * *
//   * *
//    *

const size = 3;

for (let i = size; i >= -size; i--) {
    let row = '';

    for (let j = 1; j <= Math.abs(i); j++) {
        row += ' ';
    }

    for (let k = size; k >= Math.abs(i); k--) {
        row += '* ';
    }

    console.log(row);
}

File: patterns/python-patterns/pattern84.py
# Pattern 84: ...

#    *
#   * *
#  * * *
# * * * *
#  * * *
#   * *
#    *

size = 3

for i in range(size, -size-1, -1):
    for j in range(1, abs(i)+1):
        print(" ", end="")
    for k in range(size, abs(i)-1, -1):
        print("* ", end="")
    print()
